### ISSUE TITLE
Make internal use of `keys` and `names` consistent

### ISSUE THREAD

**Issue by dpanici** [PlasmaControl/DESC#1121]
For a future PR: The variable we call `keys` here and in the other util functions below is the same as what we call `names` in the `compute` functions above and in other files. We should probably rename for consistency across the code.

_Originally posted by @ddudt in https://github.com/PlasmaControl/DESC/pull/1027#discussion_r1669324227_

**Comment by zpotthoff** [comment]
Hi @dpanici, is this issue still open? I'm happy to work on it if it is.

**Comment by dpanici** [comment]
@zpotthoff yes it is still open! feel free to open a PR and work on this issue, thanks!
